gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/hive/HiveDatasetFinder.java

```diff
@@ -181,6 +190,11 @@ protected HiveDatasetFinder(FileSystem fs, Properties properties, HiveMetastoreC
       throw new RuntimeException(e);
     }
     this.jobConfig = ConfigUtils.propertiesToConfig(properties);
+
+    String rejectedFormatSerDe = properties.getProperty(REJECTED_DATA_FORMAT_KEY, StringUtils.EMPTY);
+    Optional<HiveSerDeWrapper.BuiltInHiveSerDe> serDe =
```

Comment:
Why not support a list of rejected data formats?

Also, how about extracting out the SerDe class name as a string when an enum match is found, but if one is not found then use the string as provoided? This will allow SerDes not in BuiltInHiveSerDe to be supported.